### DISCUSSION TITLE
Can LERN be installed, configured and used standalone?

### DISCUSSION THREAD

**Discussion by manojLondhe** [Sunbird-Ed/Community#639] [Installation]
Hello everyone,

We have a potential LERN adopter, for whom use cases fit into LERN.

To avoid high server infra cost, and complexity in setting entire SB ED stack, want to know if LERN be installed, configured and used standalone?

Kindly let know if this can already be done, or if this on roadmap.

cc @vinod303 @parthlawate @coolbung

**Comment by parthlawate** [comment]
Can you share some of the example use cases @manojLondhe ?

**Comment by vinukumar-vs** [comment]
@manojLondhe As of today, there is not separate installation steps to adopt only Lern BB. Today all these services are deployed as part of Sunbird-ED installation.
Fresh installation of Sunbird-ED(release-6.0.0)
https://ed.sunbird.org/use/release/installing-ed-platform/release-6.0.0/prerequisites-for-your-own-sunbird-ed-instance

If you want to take only Sunbird-Lern BB infra & services, please read the sunbird-ED infra provisioning & deployment and you have to customise yourself for only Lern BB. 

@gandham-santhosh Please share your suggestions if any.
[@AmiableAnil](https://github.com/AmiableAnil), @Krishnaj20

**Reply by gandham-santhosh** [reply]
You can leverage Ed installation stack and instructions and setup all the components required for Lern (trigger only Lern and its dependant Jenkins jobs). You have to make changes to the Proxy and APIManager to have only Lern specific proxy configurations, API's available in the system. After the infra setup is done, you should also setup the functional configurations. 
Also, as far as I know currently Lern don't have portal. @AmiableAnil please share your view here

**Comment by ashokreddy1208** [comment]
@manojLondhe @parthlawate I hope you have gone thru the responses shared by the community. Let us know if you have any specific follow up questions. If not, please mark the thread as answered.
